litho-widget/src/main/java/com/facebook/litho/widget/CardSpec.java

```diff
@@ -89,28 +89,32 @@ static Component onCreateLayout(
     final int shadowBottom = getShadowBottom(elevation);
     final int shadowHorizontal = getShadowHorizontal(elevation);
 
-    return Column.create(c)
-        .child(
-            Column.create(c)
-                .marginPx(HORIZONTAL, shadowHorizontal)
-                .marginPx(TOP, shadowTop)
-                .marginPx(BOTTOM, shadowBottom)
-                .backgroundColor(cardBackgroundColor)
-                .child(content)
-                .child(
-                    CardClip.create(c)
-                        .clippingColor(clippingColor)
-                        .cornerRadiusPx(cornerRadius)
-                        .positionType(ABSOLUTE)
-                        .positionPx(ALL, 0)))
-        .child(
-            CardShadow.create(c)
```

Comment:
You can inline that check in this line like below instead of breaking it down into builder:
```
     .child(
         elevation > 0 ? CardShadow.create(c)......
              : null)
```

Reply:
applied inline check